lib/charms/ovn_charm.py

```diff
@@ -116,6 +116,7 @@ class BaseOVNChassisCharm(charms_openstack.charm.OpenStackCharm):
     enable_openstack = False
     bridges_key = 'bridge-interface-mappings'
     nrpe_check_services = []
+    metadata_agent = 'neutron-ovn-metadata-agent'
```

Comment:
See previous comment about handling this in the per-release class `__init__` method which would make this redundant.